<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ALKIS-Suchmodul</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>ALKIS-Suchmodul</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_bundesland">
        <property name="text">
         <string>Bundesland:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="bundesland_combo">
        <item>
         <property name="text">
          <string>Nordrhein-Westfalen</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hessen</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Niedersachsen</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rheinland-Pfalz</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Gemarkung:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Flur:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Flurstück:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="zaehler_edit">
          <property name="placeholderText">
           <string>Zähler</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nenner_edit">
          <property name="placeholderText">
           <string>Nenner (optional)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="gemarkung_edit">
        <property name="placeholderText">
         <string>Gemarkungsname</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="flur_edit">
        <property name="placeholderText">
         <string>Flurnummer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="status_label">
     <property name="styleSheet">
      <string notr="true">color: #666; font-style: italic;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="suchen_button">
       <property name="text">
        <string>Suchen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="schliessen_button">
       <property name="text">
        <string>Schließen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>bundesland_combo</tabstop>
  <tabstop>gemarkung_edit</tabstop>
  <tabstop>flur_edit</tabstop>
  <tabstop>zaehler_edit</tabstop>
  <tabstop>nenner_edit</tabstop>
  <tabstop>suchen_button</tabstop>
  <tabstop>schliessen_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
